docs/parameters.md

```diff
@@ -0,0 +1,324 @@
+# Passing parameters to brokers
+
+Table of Contents
+- [Overview](#overview)
+- [Design](#design)
+  - [Basic example](#basic-example)
+  - [Passing sensitive data](#passing-sensitive-data)
+  - [Value types](#value-types)
+    - [String](#string)
+    - [JSON object](#json-object)
+    - [Other primitive types (int, bool, float)](#other-primitive-types-int-bool-float)
+    - [Arrays](#arrays)
+  - [Passing all parameters from a single source at once](#passing-all-parameters-from-a-single-source-at-once)
+  - [Conflict resolution](#conflict-resolution)
+
+## Overview
```

Comment:
@nilebox 

I had meant sourcing the entire response from a single JSON blob in a specific key of a secret.  As far as I understand, most reusable web components that generate forms from JSONSchema just shove things into a single blob - it would be very convenient to use such a blob unmodified.  Currently in this proposal, as far as I understand, you can only set a specific parameter from such a blob, not the whole set of parameters.